app/components/__test__/Column.test.tsx

```diff
@@ -0,0 +1,24 @@
+// @vitest-environment jsdom
```

Comment:
Not sure, what might already be discussed @manuelpuchta , but I believe these kind of tests have no real value. I would only consider writing a component test when there is some logic in it.